app/src/atoms/SelectField/index.tsx

```diff
@@ -40,10 +49,25 @@ export interface SelectFieldProps {
   isSearchable?: boolean
   /** optional width to specify the width of the select field and dropdown menu */
   width?: string
+  dropdownType?: 'rounded' | 'neutral'
 }
 
+const TITLE_STYLE = css`
+  color: ${COLORS.black90};
+  padding-bottom: ${SPACING.spacing8};
+  font-size: ${TYPOGRAPHY.fontSizeLabel};
+  font-weight: ${TYPOGRAPHY.fontWeightSemiBold};
+  line-height: ${TYPOGRAPHY.lineHeight12};
```

Comment:
maybe using `isTouchscreen` would be easy?
seems that it was implemented by Brent recently.
components/src/ui-style-constants/responsiveness.ts

```suggestion
  font-size: ${TYPOGRAPHY.fontSizeLabel};
  font-weight: ${TYPOGRAPHY.fontWeightSemiBold};
  line-height: ${TYPOGRAPHY.lineHeight12};
```

Reply:
I think we can git rid of the css, use `StyledText` with `as` and `fontWeight` props, and let it handle styling for ODD